host.json

```diff
@@ -0,0 +1,27 @@
+{
+        "name": "contivModel",
+        "objects": [
+                {
+                        "name": "Bgp",
+                        "type": "object",
+                        "key": ["name"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "title": "host name",
+          "length": 256
+        },
+                                "AS": {
```

Comment:
Can we prefix AS and neighbor attributes with bgp- so that we can add other attributes to same host object?